.openpublishing.redirection.csharp.json

```
@@ -1826,6 +1826,10 @@
             "source_path_from_root": "/docs/csharp/misc/cs1007.md",
             "redirect_url": "/dotnet/csharp/language-reference/compiler-messages/overload-resolution"
         },
+        {
+            "source_path_from_root": "/docs/csharp/language-reference/compiler-messages/cs1502.md",
+            "redirect_url": "/dotnet/csharp/misc/cs1503.md"
```

Ooops. The redirection shouldn't include the `.md` extension:

```suggestion
            "redirect_url": "/dotnet/csharp/misc/cs1503"
```